### ISSUE TITLE
OpenDark RC2

### ISSUE THREAD

**Issue by pike2k** [obelisk79/OpenTheme#135]
First let me say. Thanks for your nice work!

I am finding it extremely difficult to see if the intended 'point' is selected or not in this example:

![opendark_point](https://github.com/user-attachments/assets/4d67d719-e31c-40d5-bf8a-2833b5fbbc87)

Maybe we need to add something more in FC itself to help with identifying what is selected or not?

**Comment by obelisk79** [comment]
Yeah, I'd like to reduce the amount of color used in sketcher which would help in situations like this

**Comment by pike2k** [comment]
In my example it doesn't help no matter how much you zoom in.

Ideas for FC tweaks (brainstorming):
Maybe sometime in future we could enlarge the 'vertex'? slightly when zooming in?
The red coincidents kind of "melt" into the background, at least for me. They too could enlarge a little when zooming in?
Maybe an effect could be added when selecting a 'vertex' like that it pulses or something? I don't know how other CAD software does it.
Last but not least, smarter colors. Some colors simply don't mix and match. Like the blue and green in the example. 

Maybe these are 2.0 enhancements, but I'd like to have some enhancements for 1.1 :)
